src/main/java/org/reaktivity/nukleus/kafka/internal/stream/ClientStreamFactory.java

```diff
@@ -136,7 +146,7 @@ public ClientStreamFactory(
         groupBudget = new Long2LongHashMap(-1);
         groupMembers = new Long2LongHashMap(-1);
         connectPoolFactoryConsumer.accept((networkName, ref) ->
```

Comment:
Rename `connectPoolFactoryConsumer` to `setConnectionPoolFactory`.

Reply:
done